enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -325,6 +325,8 @@ After the SDK is configured, you can embed your dashboard using the `StaticDashb
 - **withCardTitle**: `boolean` – Whether the dashboard cards should display a title.
 - **withDownloads**: `boolean | null` – Whether to hide the download button.
 - **hiddenParameters**: `string[] | null` – A list of parameters that will not be shown in the set of parameter filters. [More information here](https://www.metabase.com/docs/latest/questions/sharing/public-links#filter-parameters)
+- **onLoad**: `(dashboard: Dashboard | null) => void;` - event handler that triggers after dashboard loads, but without its cards - at this stage dashboard title, tabs and cards grid is rendered, but cards content is not yet loaded.
```

Comment:
nit: I feel `onLoad` should be reserved for when everything has been loaded. This is what I'd expect from the API. It's comparable to [load](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) and [DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event) where `load` is only fired when the whole page including its contents are loaded as opposed to the latter event where only HTML is parsed.

I'd propose `onLoad` to replace `onLoadWithCards` and come up with a new name to replace this _only dashboard has loaded_ event.